src/main/resources/flows-config.yaml

```
@@ -809,11 +809,18 @@ flow:
         condition: EnableMultipleProviders
       - name: paid-by-ccap
   multiple-providers:
+    condition: EnableMultipleProviders
     nextScreens:
       - name: confirm-provider
   confirm-provider:
+    condition: EnableMultipleProviders
     nextScreens:
+      - name: error-response-recorded
+        condition: HasProviderResponse
```

Nit: can we rename this to currentProviderHasResponse? or something like that. The current name is misleading because it sounds like a provider has responded and thats fine with multiple providers

```suggestion
        condition: HasProviderResponse
```

---

lol the suggested change... is the same thing? 😂

---

Changed the name though 👍